### ISSUE TITLE
System tag doesn't get colored with Use Servernames

### ISSUE THREAD

**Issue by Aproxia-dev** [estroBiologist/pluralchum#49]
The `Default System Tag Color` doesn't get respected when using the `Use Servernames` option, no matter what `Default System Tag Color` is set to.

**Comment by ariagivens** [comment]
I am not able to reproduce this. Are you using a servertag? There is a known limitation if you are using server specific system tags but otherwise it should work.

**Comment by Aproxia-dev** [comment]
No, these are all global tags from all systems I've seen this happen to. The member color just applies to the entire name instead of it having a different color for the system tag.

This is the expected behavior with `Use Servernames` off, none of the information is server-specific:
![image](https://github.com/estroBiologist/pluralchum/assets/53254254/9c38ac8f-d2b7-41f7-ac31-20e99c0cfb7d)

This is with `Use Servernames` on:
![image](https://github.com/estroBiologist/pluralchum/assets/53254254/1a519312-f87d-4052-9b49-39b3dcfd0590)
As you can see, the entire name gets colored with the default member color, which is unexpected behavior.

My girlfriend's been kind enough to try to run the plugin on a fresh instance of BD, and had the same issue with the latest version of Discord and BD at the time of making this issue, as well as PluralChum v2.2.1. The screenshots were taken on the current newest versions of Discord and BD with some extra themes and plugins installed.

It is also worth noting that both of us use Linux, however I don't think this would be a platform-specific issue, despite how poorly made the Linux version of Discord is.
